app/views/sessions/feed.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Feed</h1>
+
+<% unless current_user.feed.nil? %>
+  <% current_user.feed.each do |p| %>
+    <%= render p %>
+  <% end %>
+<% end %>
+<% if current_user.feed.nil? %>
```

Comment:
I would use use `unless current_user.feed`. But isn't feed an array that will always be non-nil?